src/components/dragBoxLayer.ts

```diff
@@ -31,6 +31,15 @@ export module Components {
     private _dragCallbacks: Utils.CallbackSet<DragBoxCallback>;
     private _dragEndCallbacks: Utils.CallbackSet<DragBoxCallback>;
 
+    /**
+     * Constructs a DragBoxLayer.
+     * 
+     * A DragBoxLayer is a SelectionBoxLayer with a built-in Interactions.Drag.
```

Comment:
Maybe change to 
"A Drag Box Layer is a Selection Box Layer with a built-in Drag Interaction."